crates/sui-json-rpc/src/transaction_execution_api.rs

```diff
@@ -146,12 +147,72 @@ impl TransactionExecutionApi {
             ExecuteTransactionRequest {
                 transaction: txn,
                 request_type,
-            }
+            },
+            None
         ))
         .await?
         .map_err(Error::from)?;
         drop(orch_timer);
 
+        self.handle_post_orchestration(
+            response,
+            opts,
+            digest,
+            input_objs,
+            transaction,
+            raw_transaction,
+            sender,
+        )
+        .await
+    }
```

Comment:
Worth checking if you can remove the original handler, or at least have it return an error